src/types/resource_map.rs

```diff
@@ -0,0 +1,9 @@
+use crate::types::Alias;
+use std::collections::HashMap;
+
+/// `kml:ResourceMap`, [10.13](https://docs.ogc.org/is/12-007r2/12-007r2.html#591) in the KML specification.
+#[derive(Clone, Debug, Default, PartialEq)]
+pub struct ResourceMap {
+    pub aliases: Option<Vec<Alias>>,
```

Comment:
Is there a reason for using `Option<Vec>` instead of just an empty `Vec` here? Since there isn't a wrapper element it looks like it would be equivalent

Reply:
Good question, and you're right. Done.